src/main/java/controllers/ConvertModel.java

```diff
@@ -0,0 +1,17 @@
+package controllers;
+
+import dto.UserResponseDto;
+import model.User;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ConvertModel {
+    protected UserResponseDto toDto(User user) {
```

Comment:
checklist: _Let’s create a separate private method on the controller layer to mapping User to UserResponseDto_
I think we don't need separete class

Reply:
We talked about it in QnA. Most likely, this is a valid option.